Fix build when wxUSE_BASE64=0

PBfordev opened the pull request:
Building wxWidgets with without base64 support is probably quite uncommon (I did it on accident), so feel free to reject.

Also, I am not sure whether just returning `false` from `wxSVGBitmapEmbedHandler::ProcessBitmap()` without any error message is the best thing to do. Perhaps using `wxFAIL_MSG()` as it is done in `wxBitmapBundle::FromResources()` on platforms without resource support would be a bit better.

**EDIT**
The CI log did not help me to figure out which test failed. When running test_gui locally I get a bunch of failures
<details>
<summary>Test log</summary>

```
Test program for wxWidgets GUI features
build: 3.1.7 (wchar_t,Visual C++ 1900,wx containers,compatible with 3.0)
compiled using msvc 1931 (full: 193131106)
running under Windows 10 (build 19044), 64-bit edition as petr

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
test_gui.exe is a Catch v1.12.2 host application.
Run with -? for options

-------------------------------------------------------------------------------
ClippingBoxTestCase::wxGCDC
  OneRegionRTL
-------------------------------------------------------------------------------
D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(2293)
...............................................................................

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(847):
warning:
  Skipping test because RTL layout direction is not supported on this platform

-------------------------------------------------------------------------------
ClippingBoxTestCase::wxGCDC
  OneDevRegionRTL
-------------------------------------------------------------------------------
D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(2333)
...............................................................................

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(1028):
warning:
  Skipping test because RTL layout direction is not supported on this platform

-------------------------------------------------------------------------------
ClippingBoxTestCase::wxGCDC
  OneDevRegionRTL TransformMatrix
-------------------------------------------------------------------------------
D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(2338)
...............................................................................

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(1028):
warning:
  Skipping test because RTL layout direction is not supported on this platform

-------------------------------------------------------------------------------
ClippingBoxTestCase::wxGCDC(GDI+)
  OneRegionRTL
-------------------------------------------------------------------------------
D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(2621)
...............................................................................

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(847):
warning:
  Skipping test because RTL layout direction is not supported on this platform

-------------------------------------------------------------------------------
ClippingBoxTestCase::wxGCDC(GDI+)
  OneDevRegionRTL
-------------------------------------------------------------------------------
D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(2661)
...............................................................................

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(1028):
warning:
  Skipping test because RTL layout direction is not supported on this platform

-------------------------------------------------------------------------------
ClippingBoxTestCase::wxGCDC(GDI+)
  OneDevRegionRTL TransformMatrix
-------------------------------------------------------------------------------
D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(2666)
...............................................................................

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(1028):
warning:
  Skipping test because RTL layout direction is not supported on this platform

-------------------------------------------------------------------------------
ClippingBoxTestCase::wxGCDC(Direct2D)
  OneRegionRTL
-------------------------------------------------------------------------------
D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(2950)
...............................................................................

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(847):
warning:
  Skipping test because RTL layout direction is not supported on this platform

-------------------------------------------------------------------------------
ClippingBoxTestCase::wxGCDC(Direct2D)
  OneDevRegionRTL
-------------------------------------------------------------------------------
D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(2990)
...............................................................................

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(1028):
warning:
  Skipping test because RTL layout direction is not supported on this platform

-------------------------------------------------------------------------------
ClippingBoxTestCase::wxGCDC(Direct2D)
  OneDevRegionRTL TransformMatrix
-------------------------------------------------------------------------------
D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(2995)
...............................................................................

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(1028):
warning:
  Skipping test because RTL layout direction is not supported on this platform

-------------------------------------------------------------------------------
ClippingBoxTestCase::wxSVGFileDC
  OneRegionRTL
-------------------------------------------------------------------------------
D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(3606)
...............................................................................

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(847):
warning:
  Skipping test because RTL layout direction is not supported on this platform

-------------------------------------------------------------------------------
ClippingBoxTestCase::wxSVGFileDC
  OneDevRegionRTL
-------------------------------------------------------------------------------
D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(3646)
...............................................................................

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\graphics\clippingbox.cpp(1028):
warning:
  Skipping test because RTL layout direction is not supported on this platform

-------------------------------------------------------------------------------
Grid::SelectUsingEndKey
-------------------------------------------------------------------------------
D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\controls\gridtest.cpp(951)
...............................................................................

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\controls\gridtest.cpp(982): FAILED:
  CHECK( m_grid->IsVisible(8, 9) )
with expansion:
  false

-------------------------------------------------------------------------------
wxTextCtrl::LongPaste
-------------------------------------------------------------------------------
D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\controls\textctrltest.cpp(1360)
...............................................................................

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\controls\textctrltest.cpp(1422): FAILED:
  CHECK( numLines == NUM_LINES + 2 )
with expansion:
  2 == 10002 (0x2712)

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\controls\textctrltest.cpp(1423): FAILED:
  CHECK( text->GetLineText(numLines - 1) == "THE END" )
with expansion:
   == "THE END"

-------------------------------------------------------------------------------
ExecTestCase
  TestExecute
-------------------------------------------------------------------------------
D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\exec\exec.cpp(72)
...............................................................................

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\exec\exec.cpp(243): FAILED:
  REQUIRE( stderr_arr[0].Contains("file") )
with expansion:
  false

-------------------------------------------------------------------------------
ExecTestCase
  TestAsyncRedirect
-------------------------------------------------------------------------------
D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\exec\exec.cpp(75)
...............................................................................

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\exec\exec.cpp(381): FAILED:
  REQUIRE( tis.ReadLine().Contains(expectedContaining) )
with expansion:
  false

-------------------------------------------------------------------------------
wxHtmlPrintout::Pagination
-------------------------------------------------------------------------------
D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\html\htmprint.cpp(47)
...............................................................................

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\html\htmprint.cpp(104): FAILED:
  CHECK( CountPages(pr) == 3 )
with expansion:
  4 == 3

-------------------------------------------------------------------------------
SettingsTestCase
  GetFont
-------------------------------------------------------------------------------
D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\misc\settings.cpp(33)
...............................................................................

D:\Dev\Desktop\!Lib\wxWidgets-PB\tests\misc\settings.cpp(33): FAILED:
  {Unknown expression after the reported line}
due to unexpected exception with message:
  failed to get LOGFONT in wxCreateFontFromStockObject() at D:\Dev\Desktop\
  !Lib\wxWidgets-PB\src\msw\settings.cpp:156

===============================================================================
test cases:   368 |   363 passed | 5 failed
assertions: 19828 | 19821 passed | 7 failed
```
</details>

but I cannot figure out which, if any, is related to my commits (which should not affect the default build at all).

PBfordev committed:
Fix building wxIniConfig when wxUSE_BASE64=0

Use guards for wxUSE_BASE64 just as wxConfigBase does.

PBfordev committed:
Fix building wxSVGBitmapEmbedHandler when wxUSE_BASE64=0

Use guards for wxUSE_BASE64 in wxSVGBitmapEmbedHandler::ProcessBitmap()
and just return false there when wxUSE_BASE64=0.

vadz commented:
Yes, I think using `wxFAIL_MSG()` would be preferable, as not getting anything at all in the output without any explanation would be rather confusing. If you could please update the PR to do it like this, I'd be glad to merge it. TIA!

PBfordev committed:
Show error message wxSVGBitmapEmbedHandler::ProcessBitmap()

Do not just silently fail in wxSVGBitmapEmbedHandler::ProcessBitmap()
when wxUSE_BASE64=0, use wxFAIL_MEG() to show why it failed.

PBfordev commented:
I have added an error message, hopefully understandable (however, it is not intended for a user).

As for the test failure, I ran test_gui locally and I got the same failures as for the PR branch (and a bunch more I get pseudorandomly, probably locale-related).
